### PR TITLE
Change progress bar logger to print all eval metrics

### PR DESCRIPTION
**Pull request by rishab-partha**
# What does this PR do?

Make the progress bar logger print all eval metrics instead of just accuracy.

# What issue(s) does this change relate to?

[CO-1857](https://mosaicml.atlassian.net/browse/CO-1857
)

[CO-1857]: https://mosaicml.atlassian.net/browse/CO-1857?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: composer/loggers/progress_bar_logger.py [hunk 1]
```diff
@@ -74,6 +69,7 @@ def log_data(self, data: Dict[str, Any]):
             # Check if any substring of the key matches the keys to log
             if any(key_to_log in k for key_to_log in self.keys_to_log):
                 formatted_data[k] = format_log_data_value(v)
+                #print(k)
```
Comment (eracah) [inline composer/loggers/progress_bar_logger.py:72]: remove!

**Review by eracah — COMMENTED** [review]
Can you add some sample progress bar output screenshots or gifs showing this change?

**Comment by rishab-partha** [comment]
sample progress bar output:

Screen Shot 2023-06-07 at 4 27 03 PM

**Review by eracah — APPROVED** [review]
LGTM I guess

**Pull request merged**